libs/wingsdk/src/target-sim/counter.inflight.ts

```diff
@@ -23,42 +23,58 @@ export class Counter implements ICounterClient, ISimulatorResourceInstance {
 
   public async cleanup(): Promise<void> {}
 
-  public async inc(amount: number = 1): Promise<number> {
+  public async inc(
```

Comment:
It feels more intuitive for me that the key will be the first parameter, what do you think? is it too complex to switch the order?

Reply:
I understand what you said, Revital, but I had doubts about it because currently the standard way to add 1 to the counter is `counter.inc()`, which wouldn't be affected. However, if I wanted to add more than the default value of 1, currently it is `counter.inc(2)`, and it would become something like `counter.inc(nil, 2)`.

Where the default id wouldn't need to be part of the call, but since the id is the first parameter, I would have to pass the value `nil`.